cmd/promu.go

```diff
@@ -54,8 +55,7 @@ var Promu = &cobra.Command{
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
 	if err := Promu.Execute(); err != nil {
-		fmt.Println(err)
```

Comment:
Strangely cobra was doing it by itself. Removing this line did fixed the double error display.